bin/needs-a-genny

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
+
+genny_path = ->(slug) {
+  File.join 'exercises', slug, '.meta', 'generator'
```

Comment:
here, too same comment https://github.com/exercism/ruby/pull/991#discussion_r312620485 about `exercises` forcing only being able to run it from one directory